jenkinsci/tuleap-git-branch-source-plugin
# feat: Be able to retrieve and build a PR from same the origin repository

Part of [story#18320](https://tuleap.net/plugins/tracker/?aid=18320) native support of pull requests in tuleap branch source jenkins plugin

Pull Request from fork are not supported yet.
This contribution also updates the needed dependencies

How to test:
 - Enable the `Tuleap Pull Requests from origin autodiscovery` trait in
   the configuration menu. Note: By default it should be enable when
creating a new Tuleap job
 - Save the configuration and/or launch the scan
=> You should see the pull requests jobs created of the concerned
repository
=> If a fork is detected then a message is diplayed which said that the
fork are not supported. There is no job created too.
